app/routes/docs.faq.md

```diff
@@ -58,6 +58,23 @@ Clients connecting to GCN only need to be able to make _outbound_ (egress) TCP c
   </tbody>
 </table>
 
+### How can I tell that my Kafka client is working?
+
+Subscribe to the topic `gcn.heartbeat`, which broadcasts a test message approximately once a second. The contents of the heartbeat messages looks like this:
+
+```json
+{
+  "$schema": "https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices/core/Alert.schema.json",
```

Comment:
Should we use the `.../schema/stable/gcn...` route here?

Reply:
Although, we _could_ generate that URL automatically from the @nasa-gcn/schema dependency... one moment.

Reply:
Gotcha, I didn't see your reply before my edit, but overall this all looks good

Reply:
Take a look again please.